styles.css

```diff
@@ -0,0 +1,129 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-image: url(./background1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-top: 160px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  padding: 10%;
+  text-decoration: none;
+  margin-bottom: 114px;
+  margin-right: 24px;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]** Could you increase your `toolbar` width to 100%? This way, it will take all of the horizontal display of the page!